docs/core/versions/index.md

```diff
@@ -19,9 +19,13 @@ This article aims at clarifying how the .NET Core SDK and runtime are versioned.
 
 There are lots of moving parts that version independently in .NET Core. However, starting with .NET Core 2.0, there is an easy to understand top-level version number that everybody understands to be *the* version of ".NET Core" as a whole. The rest of this document goes into the details of the versioning of all those parts. These details can be important if you're a package manager, for example.
 
+> [!IMPORTANT]
+> The versioning details explained on this topic don't apply to the current version of the .NET Core SDK and runtime.
+> The version scheme is changing in future releases. You can see the current proposal at the [dotnet/designs](https://github.com/dotnet/designs/pull/29) repository.
+
```

Comment:
It think we can explain it if we use a few more words. How about:

The .NET Core SDK and .NET Core runtime are released independently and therefore their version numbers are  independent. The first attempt at resolving this challenge was to abandon aligning their numbers and simply version them separately using semantic versioning. This resulted in considerable confusion, so the approach has been changed to return to alignment between the SDK and runtime. Since the semantic versioning approach was short lived, it is largely ignored in this document, but you will find SDK version numbers in the 2.1.n range that shipped with 2.0.5 versions of the runtime. You can see the future SDK version numbering plan at the [dotnet/designs](https://github.com/dotnet/designs/pull/29) repository.